src/EnergyPlus/SetPointManager.cc

```diff
@@ -3917,6 +3921,16 @@ namespace SetPointManager {
 						// Call the CALC routine, with an optional argument to only set
 						// the initialization NODE(:)% setpoint, and not the OutAirSetPtMgr(:)%SetPt
 						OutAirSetPtMgr( SetPtMgrNum ).calculate( NodeNum, true );
+						Node( NodeNum ).TempSetPointLo = OutAirSetPtMgr( SetPtMgrNum ).SetPtLo;
```

Comment:
@mjwitte I do not think we need to pass the NodeNum argument either. We can set to the node value right after calling "calculate" routine.